### ISSUE TITLE
Enroll only to courses in categories I'm interested in

### ISSUE THREAD

**Issue by cullzie** [aapatre/Automatic-Udemy-Course-Enroller-GET-PAID-UDEMY-COURSES-for-FREE#48]
Kind of in a similar vein to the #36 
I don't want to be enrolled in anything under `Music` category for example as I cannot sing :microphone: 

Top level categories which courses fall under:
```
Development
Business
IT & Software 
Office Productivity
Personal Development
Design
Marketing
Health & Fitness
Music
```

This list is in English of course so each user would have to define these in their own language

Leaving the option blank would add you to all courses. Otherwise you will only be enrolled in the courses which fall under the categories you have picked

**Comment by fakeid30** [comment]
so how will someone enroll in French Music classes ?

**Comment by fakeid30** [comment]
Ok this seems like a good idea.

**Comment by Vthechamp22** [comment]
Can I try working on this?

**Comment by fakeid30** [comment]
> 
> 
> Can I try working on this?

sure , go for it . Take a look at the develop branch. I will be issuing a new release soon .

**Comment by Vthechamp22** [comment]
Actually, I have already finished making the feature, I just need to update it in the README, but I have developed the feature on the master branch. When will the develop branch be ready?

**Comment by fakeid30** [comment]
@Vthechamp22 within 5 days

**Comment by shabbysinger** [comment]
Hey, I forked this repo, and I was working on this specific issue, I have solved it.
https://github.com/shabbysinger/udemy-bot-free-courses

**Comment by fakeid30** [comment]
@shabbysinger we are working on this on PR #55 

Please feel free to contribute there.

**Comment by cullzie** [comment]
Issue closed with PR #55 
Thanks @Vthechamp22
